channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpBotRequest.kt

```diff
@@ -26,6 +27,11 @@ data class JaicpBotRequest(
     override val clientId = channelUserId
     override val input: String = query ?: event ?: ""
 
+    init {
```

Comment:
Are you sure MDC propagation should be placed to the constructor of the data class?